### DISCUSSION TITLE
[Projects Beta] Organization level Issues (+labels, milestones, etc)

### DISCUSSION THREAD

**Discussion by danielBingham** [community/community#14722] [Projects and Issues]
We've had the need for organization level issues for a while and Projects Beta has really highlighted that need.  Way back in the day, pre-projects, we used to use Waffle for our board view of our repositories and already then we had issues with stories that spanned multiple repositories.  We had to settle on a single repo in the organization that was the home of our issues and link PRs across multiple other repos to issues in that repo.  That worked well when we were a single team working primarily on a single product.  We are now many teams working on several products that overlap and share code and it's no longer possible to point to any one of those repos as our "core" repo.  

What we really need are organization level issues that allow us to organize our work across repositories, products, and teams at the organization level (where it really lives) with the ability to link PRs from multiple repos to each of those issues. This would give us the overall view, would vastly help managers and PMs track their teams work and collaborate across teams, and would avoid the issue of losing issues in repos or having duplicate issues in different repos.  

There's a work around for this in the mean time, which is to define a single dummy repo as the "project management" repo, but this is a poor workaround.  Projects and discussions already exist at the organizational and team level.  Issues (and all their accompanying features) should join them.

**Comment by Greendogo** [comment]
My company uses JIRA tickets which let you create a ticket irrespective of which Repos it may be tied to.  You can, for example, go to a Jira ticket and create a branch or a PR on the appropriate repositories.  Some tickets span multiple repositories.

Extremely useful functionality which prevents the ticketing system dictating the structure of a project and the related repositories.

**Reply by lmilbaum** [reply]
We also use Jira for our internal projects. But, we also maintain an open source organization with logs of repositories.  Internal Jira is not an valid option for our use case.

**Comment by odahcam** [comment]
I was really expecting something from GitHub at this point. Enterprise ticketing systems are sometimes overkill for certain needs, just having org level issues would be a killer feature.

**Comment by CatsMiaow** [comment]
We also faced the same problem when considering moving from `Jira` to `GitHub Projects`.

When working on projects or tasks within an organization, it is often necessary to modify multiple repositories, so it is necessary to have an organization-level management function that allows issues in each repository to be grouped and managed together.

**Comment by linde-ibis** [comment]
We have the same problem. Is a dummy repo still the way to go?
